### ISSUE TITLE
 examples of the training cityscraps dataset in the ./datasets/citys dir.

### ISSUE THREAD

**Issue by Linda-L** [Tramac/Fast-SCNN-pytorch#42]
I use this code to train my own dataset of cityscrapes format. The loss becomes 0.001 after 500 epochs, but the test effect is very poor, and the trained class is not displayed. Can the author upload the examples of the training cityscraps dataset to the datasets folder（ in the ./datasets/citys dir.）?

**Comment by Tramac** [comment]
The dataset is too large to upload to github, you can download cityscapes from [here](https://www.cityscapes-dataset.com/downloads/).

**Comment by Linda-L** [comment]
alright, thank you very much
